Automattic/jetpack
# Stats Widget: Migrate legacy code into widget class

## Proposed changes:

* Partially migrates code specific to the Jetpack Stats dashboard widget into `class-jetpack-stats-dashboard-widget.php` from `modules/stats.php`. 
    * `modules/stats.php` is responsible for rendering the old Jetpack Stats dashboard (`/wp-admin/admin.php?page=stats`).

### Other information:


## Jetpack product discussion


p1HpG7-l7H-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Spin up this branch on your Jetpack site.
* Ensure that the Jetpack Stats widget on `/wp-admin/` works as expected, including the configuration functionality.
* Ensure that the legacy Jetpack Stats dashboard works as expected.
